signac/__main__.py

```diff
@@ -1098,8 +1117,7 @@ def main():
         'job_id',
         nargs='*',
         type=str,
-        help="One or more job ids. The job corresponding to a job "
-             "id must be initialized.")
+        help="One or more job ids. The jobs must be initialized.")
```

Comment:
"... The _corresponding_ jobs must be initialized." (Please also add that word to the other docstrings that you changed).